app/javascript/components/batch_update/components/App.js

```diff
@@ -120,7 +127,9 @@ class App extends React.Component {
       this.state.genes.length !== 0 ||
       this.state.keywords.length !== 0 ||
       this.state.partner !== null ||
-      this.state.tags.length !== 0
+      this.state.tags.length !== 0 ||
+      this.state.minPrice !== null ||
+      this.state.maxPrice !== null
```

Comment:
I'd delete the last two lines.  It's not stated explicitly anywhere but the difference between L106-118 and L124-132 is that the latter block only governs when the _initial_ search is kicked off (as opposed to iterating on the search with additional inputs). So only the "high value" fields should initiate, i.e. the ones that were already there.